src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetRandomCommand.cs

```diff
@@ -396,11 +396,14 @@ protected override void BeginProcessing()
                         this.ThrowMinGreaterThanOrEqualMax(min, max);
                     }
 
-                    int randomNumber = this.Generator.Next(min, max);
-                    Debug.Assert(min <= randomNumber, "lower bound <= random number");
-                    Debug.Assert(randomNumber < max, "random number < upper bound");
+                    for (int i = 0; i < Math.Max(1, this.Count); i++)
```

Comment:
Rather than need a call to `Math.Max()` you could simply initialize the value of this parameter to 1 by default.

Reply:
Done